src/backend/highlighter.js

```diff
@@ -30,7 +31,8 @@ export function highlight (instance) {
   const rect = getInstanceRect(instance)
   if (rect) {
     let content = ''
-    const name = getInstanceName(instance)
+    let name = getInstanceName(instance)
+    claissifyComponents && (name = classify(name))
```

Comment:
why are there so many `condition && statement` instead of `if (condition) statement` 😆 I think we should stop adding more of those